app/views/pages/home.html.erb

```diff
@@ -1,2 +1,41 @@
-<h1>Pages#home</h1>
-<p>Find me in app/views/pages/home.html.erb</p>
+
+<div class="navbar navbar fixed-top ">
+  <h1>Pages#home</h1>
+  <p>Find me in app/views/pages/home.html.erb</p>
+  <div class="dropdown">
+    <p>We are currenty <%= user_signed_in? ? "" : "not" %> signed in.</p>
+    <% if user_signed_in? %>
+    <p>The user email: <%= current_user.email %></p>
+  <% end %>
+    <div class="btn">
+      <%= link_to "Sign In", new_user_session_path, class: "btn-flat" %>
+    </div>
+    <div class="btn">
+      <%= link_to "Sign Up",  new_user_registration_path, class: "btn-flat" %>
+    </div>
+  </div>
+</div>
+
+<div class="container py-5 mt-10">
+  <div class="d-flex align-items-center justify-content-between mb-3">
+    <h3 class="mb-0">My Lists</h3>
+    <%= link_to "Create a equipment listing", new_equipment_listing_path, class: "btn btn-flat" %>
+  </div>
+  <div class="row ">
+    <% EquipmentListing.all.each do |equipment| %>
```

Comment:
I think you've already defined @equipments as EquipmentListing.all in the controller, so please replace
EquipmentListing.all.each  -> @equipment.each
The name of the instance variable may be changed later, so please check with Payla.